### PR TITLE
Change initial process flow for configuration.

### PR DESCRIPTION
**Pull request by eschlange**
* App now triggers session 1 on load instead of config page.
* Integration point was added for quit day specification.
* Updated routing logic in Application.js to use new flow.
* Updated formatting of views to be more readable.
* Updated session view to include date selection.

[#99282244]

### CODE REVIEW

File: src/js/Application.js [hunk 1]
```diff
@@ -81,10 +81,20 @@
                       emaService, eventService, sessionQuestionService,
                       sessionsService) {
 
+      var configurationIncompleteRouting = function() {
```
Comment (michaelwehrley) [inline src/js/Application.js]: @ericcf recommend writing this as a function declaration: `function configurationIncompleteRouting() {...`  That seems to make sense to me.

File: src/js/controllers/SessionsController.js [hunk 2]
```diff
@@ -69,7 +75,7 @@
     this.disableNext = function(choices, label) {
       return (choices.length > 0 &&
              !this.currentAnswer) ||
-             (label.indexOf('[INTEGRATION]') > -1 &&
+             (label.indexOf('[INTEGRATION') > -1 &&
```
Comment (michaelwehrley) [inline src/js/controllers/SessionsController.js]: did you mean to remove the ']'?

Comment (eschlange) [inline src/js/controllers/SessionsController.js]: Yes

File: src/partials/session.html [hunk 3]
```diff
@@ -14,11 +16,33 @@
...
+                       ng-click="session.openCessationReasonsModal()">
+                        Click here to add reason to quit
+                    </a>
+                </div>
+
+                <div ng-show="item.label.indexOf('[INTEGRATION|DATE]') > -1">
+                    <div ng-hide="session.cessationDate">
+                        <a id="cessation_date_selector"
+                           class="btn btn-info"
+                           onclick="$('#cessation_date_input').click()">
```
Comment (michaelwehrley) [inline src/partials/session.html]: Interesting.

Comment (eschlange) [inline src/partials/session.html]: Added this in order to hide the text field from view.

File: src/js/services/Session.js [hunk 4]
```diff
@@ -67,6 +67,8 @@
         case this.CESSATION_SESSION_POST:
           sessionNumber = 3;
           break;
+        default: // No Cessation Date set
+          sessionNumber = 1;
```
Comment (michaelwehrley) [inline src/js/services/Session.js]: Is a spec needed for this?

**Comment by michaelwehrley** [comment]
:eyeglasses: good to me! :+1:

**Pull request merged**